base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -57,12 +52,24 @@ pub struct ConsensusConstants {
     pub(in crate::consensus) emission_decay: f64,
     /// This is the emission curve tail amount
     pub(in crate::consensus) emission_tail: MicroTari,
-    /// This is the initial min difficulty for the difficulty adjustment
-    min_pow_difficulty: (Difficulty, Difficulty),
     /// The offset relative to the expected genesis coinbase value
     genesis_coinbase_value_offset: MicroTari,
     /// This is the maximum age a monero merge mined seed can be reused
     max_randomx_seed_height: u64,
+    /// This keeps track of the block split targets and which algo is accepted
+    /// Ideally this should count up to 100. If this does not you will reduce your target time.
+    algo_split: HashMap<PowAlgorithm, Algo_Time>,
+}
+
+// This is just a convenience  wrapper to put all the info into a hashmap per diff algo
+#[derive(Clone)]
+pub struct Algo_Time {
+    // NB this is very important to set this as 6 * the target time
+    pub max_target_time: u64,
```

Comment:
I'm assuming this is for the LWMA. Maybe it should be removed here and put into the LWMA logic. I have a feeling I was the one who moved it from there to the consensus constants, but I think calling it max_target_time sounds like it has meaning outside of just the LWMA

Reply:
Its totally there for LWMA. But we cant change it here, as its currently used as a *6 for monero and *60 for blake. So we need to keep it separate if we want the code to work for the current test net